src/codegen/compile.ml

```diff
@@ -11730,6 +11814,26 @@ and compile_prim_invocation (env : E.t) ae p es at =
   | OtherPrim "btstInt64", [_;_] ->
     const_sr (SR.UnboxedWord64 Type.Int64) (Word64.btst_kernel env)
 
+  | OtherPrim "setCandidLimits", [e1; e2; e3] ->
+    SR.unit,
+    compile_exp_as env ae (SR.UnboxedWord32 Type.Nat32) e1 ^^
+    Serialization.Registers.set_value_numerator env ^^
+    compile_exp_as env ae (SR.UnboxedWord32 Type.Nat32) e2 ^^
```

Comment:
Should we maybe limit the numerator below `u32::MAX` to avoid overflow when adding max bias (pls. see comment above)?

Reply:
Oh, I now saturate the bound u64::MAX instead....